cli/script/command-executor.ts

```diff
@@ -623,7 +653,7 @@ function logout(command: cli.ICommand): Promise<void> {
 function formatDate(unixOffset: number): string {
     var date: moment.Moment = moment(unixOffset);
     var now: moment.Moment = moment();
-    if (now.diff(date, "days") < 30) {
+    if (Math.abs(now.diff(date, "days")) < 30) {
```

Comment:
It works as far as I can tell. `fromNow` and `toNow` are only different based on the sign inversion.